e2e/api/backend_test.go

```diff
@@ -243,9 +242,7 @@ func (s *APIBackendTestSuite) TestResetChainData() {
 	s.StartTestBackend(params.RinkebyNetworkID)
 	defer s.StopTestBackend()
 
-	ctx, cancel := context.WithTimeout(context.Background(), DefaultNodeSyncTimeout)
-	defer cancel()
-	s.Nil(EnsureNodeSync(ctx, s.Backend.NodeManager()), "cannot ensure node synchronization")
+	s.NoError(EnsureNodeSync(s.Backend.NodeManager()), "cannot ensure node synchronization")
```

Comment:
What do you think of just throwing panic instead of returning `error`? That's just a test helper, why not keep it simple and panic?